supabase/functions/oauth/access-token.ts

```diff
@@ -81,7 +79,7 @@ export async function accessToken(req: Record<string, any>) {
         );
     }
 
-    const defaultContentType = Object.keys(headers).some(h => h.toLowerCase() == 'content-type') ? {} : { 'content-type': 'application/json' };
+    const defaultContentType = Object.keys(headers).every(h => h.toLowerCase() != 'content-type') && { 'content-type': 'application/json' };
```

Comment:
Just because this reads more like the sentence: "If there is already a `content-type` header, we don't have a default, otherwise we have a default.

Instead of this sentence: If all of the keys in the headers are _not_ `content-type`, then we add a default `content-type`.

I find the former sentence easier to follow.

```suggestion
    const defaultContentType: Record<string, string> = Object.keys(headers).some(h => h.toLowerCase() == 'content-type') ? {} : { 'content-type': 'application/json' };
```

Reply:
Done